src/app/main/component/events/services/events.service.ts

```
@@ -1,208 +1,89 @@
-        const { onlineLink, place, coordinates } = value.placeOnline;
-
-        const dateObject = new Date(date);
-
-        if (isNaN(dateObject.getTime())) {
-          return;
+  prepareForSumbit(eventInformation, dates, images, id, isUpdating): FormData {
+    let sendEventDto = {
+      ...eventInformation,
+      datesLocations: dates.map((item) => {
+        if (!item.coordinates.latitude && !item.coordinates.longitude) {
+          delete item.coordinates;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)